actions/send.server.info.email.yaml

```diff
@@ -1,98 +1,116 @@
 ---
-description: Email notifications for error emails
+description: Sends email notifications to VM owners based on a given query.
 enabled: true
 entry_point: workflows/send.server.info.email.yaml
-name: send.server.email
+name: send.server.info.email
 parameters:
   timeout:
     default: 99999
+  cloud_account:
+    description: "The clouds.yaml account to use whilst performing this action"
+    type: string
+    default: ""
+    required: true
   query_preset:
-    description: "name of a preset query - e.g server_shutoff: list all servers shutoff "
+    description: "Name of a preset query - e.g server_shutoff: list all servers shutoff"
     type: string
+    default: "server_older_than"
+    enum:
+      - "id_in"
+      - "id_not_in"
+      - "name_in"
+      - "name_not_in"
+      - "name_contains"
+      - "name_not_contains"
+      - "server_shutoff"
+      - "server_error_and_shutoff"
+      - "server_error"
+      - "server_older_than"
+      - "server_younger_than"
     required: true
```

Comment:
So we should have an optional property `project` which limits this workflow to a project.

I'd also like to put in guards that any of these cannot be ran without a project specified:
```
      - "id_in"
      - "id_not_in"
      - "name_in"
      - "name_not_in"
      - "name_contains"
      - "name_not_contains"
```

Especially the not_contains could result in 49,995 emails being sent because 5 VMs didn't match a name....